src/Event/ContaoFrameworkBootEvent.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+/**
+ * This file is part of Contao.
+ *
+ * Copyright (c) 2005-2015 Leo Feyer
+ *
+ * @license LGPL-3.0+
+ */
+
+namespace Contao\CoreBundle\Event;
+
+use Symfony\Component\DependencyInjection\ParameterBag\ParameterBag;
+use Symfony\Component\EventDispatcher\Event;
+use Symfony\Component\HttpFoundation\Request;
+use Contao\CoreBundle\Adapter\ConfigAdapter;
+
+/**
+ * Class ContaoFrameworkBootEvent
+ *
+ * @author Dominik Tomasi <https://github.com/dtomasi>
+ */
+class ContaoFrameworkBootEvent extends Event
+{
+    /**
+     * @var ParameterBag
+     */
+    protected $parameters;
+
+    /**
+     * @var Request
+     */
+    protected $request;
+
+    /**
+     * @var ConfigAdapter
+     */
+    protected $config;
+
+    /**
+     * Create a ParameterBag
+     */
+    public function __construct()
+    {
+        $this->parameters = new ParameterBag();
+    }
+
+    /**
+     * Set the Request
+     *
+     * @param Request $request
+     *
+     * @return $this
+     */
+    public function setRequest(Request $request)
+    {
+        $this->request = $request;
+
+        return $this;
+    }
+
+    /**
+     * Get the Request
+     *
+     * @return Request
+     */
+    public function getRequest()
+    {
+        return $this->request;
+    }
+
+    /**
+     * Set the Config
+     *
+     * @param ConfigAdapter $config
+     *
+     * @return $this
+     */
+    public function setConfig(ConfigAdapter $config)
```

Comment:
I don't really like this because it's not needed, right? Any listener can just get it injected :-)